tests/system/feature_store/test_feature_store.py

```diff
@@ -4818,6 +4820,45 @@ def test_merge_different_number_of_entities(self):
         ).to_dataframe()
         assert_frame_equal(expected_all, df, check_dtype=False)
 
+    @pytest.mark.parametrize("local", [True, False])
+    def test_attributes_in_offline_target(self, local):
```

Comment:
It's not specific to offline target. Also, do we need to add a test or can we add this check elsewhere?

Reply:
rename test.

I still think it should be a separated test that target this specific problem:)